plugin/src/main/java/org/opensearch/sql/plugin/rest/RestQuerySettingsAction.java

```diff
@@ -44,8 +44,12 @@
   private static final String SQL_SETTINGS_PREFIX = "plugins.sql.";
   private static final String PPL_SETTINGS_PREFIX = "plugins.ppl.";
   private static final String COMMON_SETTINGS_PREFIX = "plugins.query.";
+  private static final String LEGACY_SQL_SETTINGS_PREFIX = "opendistro.sql.";
+  private static final String LEGACY_PPL_SETTINGS_PREFIX = "opendistro.ppl.";
+  private static final String LEGACY_COMMON_SETTINGS_PREFIX = "opendistro.query.";
```

Comment:
Is this not needed? Because previously there was no setting name prefixed with `opendistro.query`?

Reply:
`opendistro.query.size_limit`

Reply:
Got it. Thanks!